tests/threading/lib

```diff
@@ -0,0 +1 @@
+../bindings/lib
```

Comment:
Unfortunately Windows doesn't support symlink so we cannot do that :(
I'm going to fix this after merging the PR by making the scons rule create a symlink on posix and a reparse point (that's basically a symlink, but different because Windows sucks...) on windows pointing on a lib folder shared among tests